plutus-benchmark/lists/bench/Bench.hs

```diff
@@ -55,5 +54,5 @@ main :: IO ()
 main = do
   -- Run each benchmark for at least 15 seconds.  Change this with -L or --timeout.
   config <- getConfig 15.0
-  evalCtx <- evaluate $ force mkEvalCtx
```

Comment:
I moved that `force` to the creation site, so that we can remove it from lots of call sites (which was risky to begin with, because we'd occasionally forget to add a `force`, thus distorting the benchmarking results).